tests/gui/components/svg-icon.spec.js

```diff
@@ -72,6 +72,17 @@ describe('Browser: SVGIcon', function () {
       m.chai.expect(element.children().attr('src')).to.equal(imgData)
     })
 
+    it('should use an empty src if there is a parsererror', function () {
+      // The following is invalid, because there's no closing tag for `foreignObject`
+      const iconContents = '<svg><foreignObject></svg>'
+      const imgData = `data:image/svg+xml,`
+      $rootScope.iconContents = iconContents
+
+      const element = $compile('<svg-icon path="iconContents">Resin.io</svg-icon>')($rootScope)
```

Comment:
Yeah, that's what I thought. So why is variable-interpolation not needed here, when it is needed on line 88?
(i.e. to clarify: I'm confused why this line _isn't_ doing variable interpolation - sorry for not being clearer in my earlier comment.)